CONTRIBUTING.md

```diff
@@ -10,6 +10,8 @@ Create a new directory if no existing one fits.
 
 Open a pull request for review.
 
+For help submitting, please review our [Git Commits for Non-Git Users](/GitCommitsForNonGitUsers.md) guide.
```

Comment:
This filename is crazy. How about just inlining the contents of that file into `CONTRIBUTING.md`.